Source/Core/MapboxApi.js

```diff
@@ -49,7 +49,7 @@ define([
         }
 
         if (!defined(errorCredit)) {
-            errorCredit = new Credit(errorString);
+            errorCredit = new Credit(errorString, undefined, undefined, false);
```

Comment:
We should probably change `Credit` to take an options object.  Any call that requires passing `undefined` is usually a sign you need one.

Reply:
I avoided doing that so I didn't have to go through the whole deprecation process.  Can I just throw in another breaking change?

Reply:
No, the right thing to do is to turn it into an object and deprecate.  It should be straightforward to do.